Automattic/vip-go-mu-plugins
# Update/wp jquery update test plugin

## Description

There have been a few tweaks lately so upgrading this plugin to the latest from the upstream (https://github.com/WordPress/wp-jquery-update-test/tree/2.0)

NOTE - we have local mods b/c the latest 2.0 breaks on read-only file systems due to capability check changes. I've opened an issue here: https://github.com/WordPress/wp-jquery-update-test/issues/14

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Add `define( 'VIP_ENABLE_TEST_JQUERY_UPDATES', true );` to your wp-config.php or config/vip-config.php file
1. Go to `wp-admin` > `Plugins` > `Test jQuery Updates`
1. Switch the version to the one shown (1.12.4) (should save correctly)
1. Check jQuery version on frontend - `jQuery.fn.jquery` in browser console, should match above.
1. Test disabling jQuery migrate in settings - then check frontend to see if jQuery migrate no longer loads (in browser console)
